src/docfx/build/BuildOptions.cs

```diff
@@ -52,7 +52,7 @@ internal class BuildOptions
             }
         }
 
-        private static CultureInfo CreateCultureInfo(string locale)
+        public static CultureInfo CreateCultureInfo(string locale)
```

Comment:
nit: move to `LocalizationUtility` now that it is shared by multiple components.